fabric-renderer-registries-v1/src/main/resources/fabric-renderer-registries-v1.mixins.json

```diff
@@ -3,11 +3,6 @@
   "package": "net.fabricmc.fabric.mixin.client.renderer.registry",
```

Comment:
You can drop this file entirely (and remove it from the mod .json as well)

Reply:
Done.